src/juju/reducers.js

```diff
@@ -16,14 +16,15 @@ export default produce(
             uuid: model.model.uuid
           };
         });
-        draftState.models.items = modelList;
+        draftState.models = modelList;
         break;
       case actionsList.updateModelStatus:
         const payload = action.payload;
-        draftState.models.items[payload.modelUUID].status = payload.status;
+        draftState.modelStatuses[payload.modelUUID] = payload.status;
     }
   },
   {
-    models: { items: {} }
+    models: {},
```

Comment:
It could be convenient to retain `items` if you do need to store some other details for models, but I guess you can always add this back in.